roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -13,7 +13,7 @@ openshift_logging_elasticsearch_proxy_image: "{{ l2_os_logging_proxy_image }}"
 openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_nodeselector | default('') }}"
 openshift_logging_elasticsearch_cpu_limit: null
 openshift_logging_elasticsearch_cpu_request: "{{ openshift_logging_es_cpu_request | default('1000m') }}"
-openshift_logging_elasticsearch_memory_limit: "{{ openshift_logging_es_memory_limit | default('1Gi') }}"
```

Comment:
That is, I'm not sure that this change will actually work - not sure what is the relationship between openshift_logging_es_memory_limit and openshift_logging_elasticsearch_memory_limit

Reply:
`openshift_logging_es_memory_limit` comes from the `openshift-logging` role [1]. So, if `openshift_logging_elasticsearch_memory_limit` isn't set, it will take what that value is.

[1] https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/defaults/main.yml#L107